cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts.adoc

```diff
@@ -209,14 +211,14 @@ $ cat << EOF | oc apply -f -
 EOF
 ----
 
-[id="rosa-mobb-cloudwatch-sts-check-aws"]
-== Check AWS CloudWatch for logs
+[id="cloud-experts-rosa-cloudwatch-sts-check-aws"]
+== Checking CloudWatch for logs
 
-* Use the AWS console or CLI to validate that there are log streams from the cluster:
+* Use either the AWS console or the AWS CLI to validate that there are log streams from the cluster:
```

Comment:
I think you need one more sentence, like: To validate the logs in the AWS CLI, run the following command: